RELEASE.md

```diff
@@ -4,11 +4,39 @@
 
 ### Master
 
-The HEAD of master is consider the most active development, to which, new features and bug fixes are applied. The expectation is that master will always build and pass tests. Development is incremental. It is expected that pull-requests are either complete, are not fully integrated into code execution path or have a feature switch which must be enabled until code completion is reached. HEAD on master is expected to be in a state that it could be released. The next major release will be tagged and released from master. If the current released version is `v0.3.1` and the next major release is `v0.4.0` when it is deemed appropriate `v0.4.0` will be tagged off master, followed by a release. From that tag work will continue on master. IF it is necessary in the future to have a patch release for `v0.4.1`, a branch will be created off the `v0.4.0` tag with the nomenclature of `releases\{major.minor}`, in this example `releases\0.4`. Branches with `releases/*` are protected branches in the repository.
+The `HEAD` of `master` branch is considered the most active development, to which new features and bug fixes are applied. The expectation is that master will always build and pass tests.
+Development is incremental. It is expected that pull-requests are either:
+ - complete and fully integrated into code execution path, or
+ - have a feature switch which must be enabled until code completion is reached.
+
+`HEAD` on master is expected to be in a state that it could be released.
+The next major release will be tagged and released from master.
+If the current released version is `v0.3.1` and the next major release is `v0.4.0`, then when it is deemed appropriate `v0.4.0` will be tagged off master, followed by a release.
+From that tag work will continue on master.
+
+*If* it is necessary in the future to have a patch release for `v0.4.1`, a branch will be created off the `v0.4.0` tag with the nomenclature of `releases/{major.minor}`, in this example `releases/0.4`.
+Branches matching `releases/*` are protected branches in the repository.
+
```

Comment:
it could be useful to add that the `HEAD` of a release branch is also expected to be releasable and it's purpose is for bug fixes only.  No new features are to be added to `release` branch.